<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserAdd</class>
 <widget class="QWidget" name="UserAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить</string>
  </property>
  <widget class="QTextBrowser" name="textBrowserUserAdd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>311</width>
     <height>331</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline;&quot;&gt;Добавление пользователя&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLastNameAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumberPhoneAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер телефона</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFirstNameAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUserAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFirstNameAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLastNameAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumberPhoneAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
